ci-operator/templates/openshift/installer/cluster-launch-installer-openstack-e2e.yaml

```diff
@@ -300,6 +300,9 @@ objects:
         export SSH_PUB_KEY=$(cat "${SSH_PUB_KEY_PATH}")
         export PULL_SECRET=$(cat "${PULL_SECRET_PATH}")
 
+        # set default if the value is not provided
+        OPENSTACK_FLAVOR="${OPENSTACK_FLAVOR:-m1.s2.xlarge"}
```

Comment:
Don't we need 'export' here?

Reply:
I don't think so, this variable is not used outside of the script. We just need to inject this value into install-config.yaml and nothing more.

Reply:
ack on the export. So there is an issue with quoting here though?